### PR TITLE
Check for b3 headers if the traceparent is not present in headers

### PR DESCRIPTION
**Pull request by rakesh-garimella**
**Description**

Changes proposed in this pull request:

- With the new changes to the  [tracing](https://github.com/kyma-project/kyma/issues/16299), the [tests](https://github.com/kyma-project/kyma/pull/16372) were adapted to use the `traceparent` header. However during the upgrade tests it can be a case that this is header is not present (as upgrade test try to upgrade from released version to main branch). In such a case try to look for `x-b3-traceid`.

**Related issue(s)**

https://github.com/kyma-project/kyma/issues/16430

### CODE REVIEW

**Review by pbochynski — DISMISSED** [review]

File: tests/fast-integration/eventing-test/eventing-test.js [hunk 1]
```diff
@@ -99,6 +112,11 @@ describe('Eventing tests', function() {
 
   // eventingTestSuite - Runs Eventing tests
   function eventingTestSuite(backend, isSKR, testCompassFlow=false) {
+    // This code can be removed when we have release 2.10
+    it('redeploys the function to use the latest code', async function() {
+      await redeployFunction(testNamespace);
```
Comment (mfaizanse) [inline tests/fast-integration/eventing-test/eventing-test.js:117]: Can we confirm if we still need to redeploy the lastorder func?

**Review by hisarbalik — DISMISSED** [review]

File: tests/fast-integration/eventing-test/eventing-test.js (hunk 1 above)
Comment (rakesh-garimella) [reply tests/fast-integration/eventing-test/eventing-test.js]: we dont need to redeploy

**Review by mfaizanse — APPROVED** [review]

**Review by hisarbalik — APPROVED** [review]

**Review by lindnerby — APPROVED** [review]

**Pull request merged**